src/App.tsx

```diff
@@ -1,60 +1,123 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
+
+function getUserByID(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserByID(todo.userId),
+}));
 
 export const App = () => {
+  const [preparedTodo, setPreparedTodo] = useState(todos);
+  const [TodoTitle, setTodoTitle] = useState('');
+  const [isPressed, setIsPressed] = useState(false);
```

Comment:
```suggestion
  const [isAdded, setIsAdded] = useState(false);
```
Incomprehensive naming.